src/classes/DMLManager.cls

```diff
@@ -105,6 +109,26 @@ public class DMLManager {
 					else if (dmlOperation == Operation.OP_DELETE){delete objList;}
 	}
 	
+	/**
+	 * Checks for CRUD on the master & merge objects and throws error if they don't have the proper object permission
+	 * If all checks pass, the merge operation is performed.
+	 * 
+	 * @param masterObj SObject the data will be merged into
+	 * @param mergeObj SObject that will be merged into masterObj
```

Comment:
doc uses the wrong property name; it's mergeList and it's a list, not a single object